hubspot/write.go

```diff
@@ -31,7 +31,10 @@ func (c *Connector) Write(ctx context.Context, config common.WriteParams) (*comm
 		write = c.Client.Post
 	}
 
-	json, err := write(ctx, url, config.ObjectData)
+	data := make(map[string]interface{})
+	data["properties"] = config.ObjectData
```

Comment:
Can you add a comment that Hubspot requires all fields to be nested within "properties"?